third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -93,20 +92,14 @@ blink::WebContentSettingsClient* GetContentSettingsClientFor(
 BraveFarblingLevel GetBraveFarblingLevelFor(ExecutionContext* context,
                                             BraveFarblingLevel default_value) {
   BraveFarblingLevel value = default_value;
-  // This is safe to call with a null pointer.
-  blink::WebContentSettingsClient* settings =
-      GetContentSettingsClientFor(context);
-  if (settings)
-    value = settings->GetBraveFarblingLevel();
+  if (context)
+    value = brave::BraveSessionCache::From(*context).GetBraveFarblingLevel();
   return value;
 }
 
 bool AllowFingerprinting(ExecutionContext* context) {
-  blink::WebContentSettingsClient* settings =
-      GetContentSettingsClientFor(context);
-  if (settings)
-    return settings->AllowFingerprinting();
```

Comment:
is `blink::WebContentSettingsClient::AllowFingerprinting` still used somewhere?

Reply:
No! Removed in https://github.com/brave/brave-core/pull/16264/commits/f6afd018f9e9bc3df182ff93b232e1b752ae85ed